xwiki-rendering-macros/xwiki-rendering-macro-message/src/test/resources/macromessage5.test

```diff
@@ -0,0 +1,24 @@
+.runTransformations
+.#-----------------------------------------------------
+.input|xwiki/2.0
+.# Validate that the paragraph is stripped when it's not needed
+.#-----------------------------------------------------
+{{info}}Only **simple** message{{/info}}
```

Comment:
You don't need bold to validate this test. You just need a single word actually for a minimal test

Reply:
Yes please. My POV is that:
* a test should test what to says it's testing
* it should be as simple as possible since otherwise it raises question. In this case, if I'm reading this test, I'm asking myself why is there a bold element and why is that important for the test.

Do you agree?

Reply:
As I really want to test here the behaviour both for the simple paragraph and what happens with styling in it (not tested elsewhere), I propose a middle-way: I keep this test with a more complete comment, and I add a new simpler one, just for the paragraph.

Reply:
> As I really want to test here the behaviour both for the simple paragraph and what happens with styling in it (not tested elsewhere),

Why? We don't any code in the message macro that has any impact on that? Am I missing something?

Reply:
> We don't have any code in the message macro that has any impact on that? Am I missing something?

Nop, just me starting to be a bit paranoïd with macro message, and wanted to cover all cases :) 
OK I'll remove it.

Reply:
> Nop, just me starting to be a bit paranoïd with macro message, and wanted to cover all cases :)

Sure but it's impossible to do that ;)

Did you also test with a table inside? or a Group Block ? or ... :)

Reply:
In short we can't redo all tests that exist on wiki markup